virttest/test_setup.py

```diff
@@ -243,22 +244,20 @@ def __init__(self, test, params):
             base_dir = f[0]
             if f[2]:
                 for name in f[2]:
-                    f_dir = os.path.join(base_dir, name)
-                    with open(f_dir, 'r') as param_f:
-                        parameter = param_f.read()
-                    logging.debug("Reading path %s: %s", f_dir,
-                                  parameter.strip())
+                    f_dir = utils_misc.SysFS(os.path.join(base_dir, name),
+                                             session=self.session)
+                    parameter = str(f_dir.sys_fs_value).strip("[]")
```

Comment:
`f_dir.sys_fs_value` might be like "[always] defer defer+madvise madvise never", then this line would not work well. I suggest to put `parameter = str(f_dir.sys_fs_value)` here and keep the if block in L255.

Reply:
okay, fdir.sys_fs_value would be something like `[always]`, so I jjust strip [] to get `always`

Reply:
Yeah, you are right. Let's keep it.